colors/dracula.vim

```diff
@@ -332,6 +332,21 @@ hi! link cssUnitDecorators DraculaPink
 hi! link cssProp DraculaCyan
 hi! link cssPseudoClassId DraculaGreenItalic
 
+"}}}2
+" Git Commit: {{{2
+
+" These groups appear when editing commit messages.
+" They are not part of the Diff interface of vim diff
+
+" The following two are misnomers. Colors are correct.
+hi! link diffFile DraculaGreen
+hi! link diffNewFile DraculaRed
+
+hi! link diffLine DraculaPurpleBold
+hi! link diffLine DraculaCyanItalic
```

Comment:
Any reason this looks suspiciously like copy paste (by which I mean, why highlight link the same group twice) ?

Reply:
Good catch... I was checking a few options to see what looked best and forgot to prune away the ones that didn't end up making the cut.